### PR TITLE
7648 PSU Thermal issue fix.

### PR DESCRIPTION
**Pull request by KrishnaKolakaluri**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -13,7 +13,7 @@ endif
 include $(ONL)/make/config.mk
 
 # All available architectures.
-ALL_ARCHES := amd64 powerpc armel arm64
+ALL_ARCHES := amd64 armel
```
Comment (jnealtowns) [inline Makefile]: I'm sure this was a temporary local change for your development but you cannot submit this change.

Comment (KrishnaKolakaluri) [reply Makefile]: Yes, that was my local change.. Didn't know that it was part of the commit, will revert it back. Thanks for pointing it out.

**Comment by KrishnaKolakaluri** [comment]
Thanks again for reviewing and i just reverted the Makefile change.

**Comment by jnealtowns** [comment]
Thanks!

**Pull request merged**